mbed_devices/_internal/htm_file.py

```diff
@@ -0,0 +1,102 @@
+"""Facilitates parsing MBED.HTM files found on the Mbed devices.
+
+For legacy reasons, we support many flavours of MBED.HTM files. The list of examples below is not exhaustive,
```

Comment:
It seems a bit fragile to rely on this HTM file if the format is subject to change at any time and we have to keep supporting the legacy formats. I suppose there's no other, more reliable, way to gather this information?

Reply:
If there is, I'm all ears. We're going to narrow it down to just parsing out `target_id` since I don't think `daplink_*` will be required.

Reply:
I don't know if there is. I think this is _probably_ okay for now, since we're narrowing it down to just parsing `target_id`.